operator/cmd/mesh/operator-common.go

```diff
@@ -33,6 +33,8 @@ type operatorCommonArgs struct {
 	tag string
 	// operatorNamespace is the namespace the operator controller is installed into.
 	operatorNamespace string
+	// watchedNamespaces is the namespaces the operator controller watches, could be namespace list separated by comma.
+	watchedNamespaces string
 	// istioNamespace is the namespace Istio is installed into.
 	istioNamespace string
```

Comment:
Replace comment with:
// istioNamespace is deprecated, use watchedNamespaces instead.

Reply:
done.

Reply:
done